rules/plists.bzl

```diff
@@ -132,17 +127,10 @@ def _substitute_plist_vars(name, plist, xcconfig):
     Returns:
         The name of the substituted plist
     """
-
-    variables = {k: v for (k, v) in xcconfig.items() if types.is_string(v)}
-
-    if len(variables) == 0:
```

Comment:
I added this check because otherwise we generated substituted plists when its not required.

Reply:
Oh i guess its no longer possible because of `select`